main/src/test/java/org/hobsoft/hamcrest/compose/ComposeMatchersTest.java

```diff
@@ -147,6 +147,14 @@ public void composeWithMatchersClonesList()
 		assertThat(actual.matches("y"), is(true));
 	}
 	
+	@Test
+	public void composeWithMatchersListReturnsMatcher()
```

Comment:
Looks like a duplicate of `composeWithMatcherListReturnsMatcher`? Be worth replacing with a `composeWithSupertypeMatcherListReturnsMatcher` test.